CHANGELOG.unreleased.md

```diff
@@ -7,6 +7,8 @@
 
 > Users must be able to say: “Nice enhancement, I'm eager to test it”
 
+- [Kubernetes recipe] Add the possibility to choose the version for the cluster (PR [#6880](https://github.com/vatesfr/xen-orchestra/pull/6880))
```

Comment:
Where the feature request come from?

Reply:
It came from https://github.com/vatesfr/xen-orchestra/issues/6842.

Reply:
Also add it to the changelog entry:

`... #6842 (PR ...)`